encrypted-media-respec.html

```diff
@@ -3611,6 +3611,12 @@
                 <li><p>Data, including <a def-id="distinctive-identifiers"></a> and <a def-id="distinctive-permanent-identifiers"></a>, transmitted between the client and server can be viewed and/or modified by other entities. See <a href="#user-tracking">User Tracking</a>.</p></li>
               </ul>
             </li>
+            <li>
```

Comment:
I don't fully understand the intent of this addition.  It doesn't seem related to browsing profiles, but perhaps does provide an additional threat that warrants secured origins.  Is that the purpose?